### DISCUSSION TITLE
Quasar Electron to Website on Linux Server

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#9881] [CLI - SSR mode] [answered]
I'm a bit new to the Quasar family. I've been using it for about 6 months now, mainly to only develop Electron Desktop apps. However I created an application recently that was for the desktop, and now I want to take the same codebase and stick it onto a Linux server, I'm running that is accessible only within our network, so it's accessible in the browser like a website.

Is there a tutorial or information someone could pass along that would explain how I could get this installed so users could use the app within their browser. I'm not sure where to begin. I'm assuming it's probably pretty easy to do.

**Comment by customautosys** [comment]
Have you tried deploying to SPA? Is there any electron specific functionality you're using?

**Reply by JokerMartini** [reply]
I have not tried that yet. I was looking for directions on that.

If i answer yest to your Electron question, what does that change?

**Reply by JokerMartini** [reply]
so i did the SPA build, is it as easy as just copying the files to my linux server from the dist/spa folder?

**Reply by customautosys** [reply]
Yes. Try that and see if anything breaks. It usually works unless you use any electron specific functionality (e.g. $q.electron.*) which would not work in SPA, or you are relying on a native module (which cannot run on the web).

After that, your next adventure can be PWA/SSR 😉

By the way, my website is a Quasar SPA: https://customautosys.com

**Reply by JokerMartini** [reply]
I do get electron errors and the page is simply just black. I'm not quite sure what is wrong. I guess I could run a local d3v build in spa until I get it working. Do you know how to fix stuff like this @siauderman ? I wonder if it's as simple as removing a few features that used node.js.

**Reply by customautosys** [reply]
You will need to remove all the electron functionality by checking process.env.MODE. You need to run quasar dev, use a debugger (probably Chrome) and fix the errors one by one.

**Comment by MilosPaunovic** [comment, marked as answer]
You should begin at the [official docs](https://quasar.dev/quasar-cli/developing-spa/deploying).
